app/src/main/res/layout/activity_webview.xml

```diff
@@ -1,5 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
-<WebView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/webview"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Rather than setting visibility it might be cleaner if you use a [ViewFlipper](https://developer.android.com/reference/android/widget/ViewFlipper).  We already use it in the MobileAppIntegrationFragment.